### PR TITLE
CTCP-590 changed indent to dashes and load rules from csv

### PR DESCRIPTION
**Pull request by crystalmark**

### CODE REVIEW

**Review by drnaylor-oc — DISMISSED** [review]
The python looks reasonable - I'll leave it to content to give their okay on that one.

**Review by PeterBandura — APPROVED** [review]

**Comment by drnaylor-oc** [comment]
@crystalmark @ardwalker One of the commits isn't signed so merging is blocked, that'll need to be fixed before this can be merged.

**Review by PeterBandura — DISMISSED** [review]

**Review by PeterBandura — APPROVED** [review]

**Review by drnaylor-oc — APPROVED** [review]

**Pull request merged**
